app/assets/javascripts/template_invocation.js

```diff
@@ -96,7 +96,7 @@ function delayed_refresh(url, data){
         url: url,
         data: data,
         dataType: "script",
-        error: function() { $("div.terminal div.printable").append(__('<div class="line error">There was an error while updating the status, try <a href="javascript:window.location.href=window.location.href">refreshing</a> the page</div>')) }
+        error: function() { $("div.terminal div.printable").append(__('<div class="line error">There was an error while updating the status, try refreshing the page.</div>')) }
```

Comment:
Updated, but FWIW, there's certainly a bunch of strings in foreman core with HTML not extracted out of the string.